CMake/fletch-tarballs.cmake

```diff
@@ -105,13 +105,13 @@ set(Eigen_dlname "eigen-${Eigen_version}.tar.gz")
 list(APPEND fletch_external_sources Eigen)
 
 # OpenCV
-# Support 2.4.11 and 3.1 optionally
+# Support 2.4.13 and 3.1, and 3.3 optionally
 if (fletch_ENABLE_OpenCV OR fletch_ENABLE_ALL_PACKAGES)
-  set(OpenCV_SELECT_VERSION 3.1.0 CACHE STRING "Select the  version of OpenCV to build.")
-  set_property(CACHE OpenCV_SELECT_VERSION PROPERTY STRINGS 2.4.11 3.1.0)
+  set(OpenCV_SELECT_VERSION 3.3.0 CACHE STRING "Select the  version of OpenCV to build.")
+  set_property(CACHE OpenCV_SELECT_VERSION PROPERTY STRINGS 2.4.13 3.1.0 3.3.0)
 
   # Expose optional contrib repo when enabling OpenCV version >= 3.x
-  if (OpenCV_SELECT_VERSION VERSION_EQUAL 3.0.0 OR OpenCV_SELECT_VERSION VERSION_GREATER 3.0.0)
+  if (OpenCV_SELECT_VERSION VERSION_GREATER_EQUAL 3.0.0)
```

Comment:
VERSION_GREATER_EQUAL is a very new CMake construct. It is not supported in versions fletch claims to support. I don't know when it came in, but likely it is in the 3.8 range. Sorry, by we need to use the clunkier
    # Just picking some version above what we support. Not less than 2.5 means version 3.0 or greater :(
    if (NOT OpenCV_SELECT_VERSION VERSION_LESS 2.5)